src/main/java/org/aksw/simba/lemming/metrics/single/SingleValueMetric.java

```diff
@@ -1,22 +1,61 @@
-package org.aksw.simba.lemming.metrics.single;
-
-import org.aksw.simba.lemming.ColouredGraph;
-import org.aksw.simba.lemming.metrics.Metric;
-
-/**
- * A metric that generates a single double value.
- * 
- * @author Michael R&ouml;der (roeder@informatik.uni-leipzig.de)
- *
- */
-public interface SingleValueMetric extends Metric {
-
-    /**
-     * Applies the metric to the given graph.
-     * 
-     * @param graph
-     *            the graph for which the metric should be calculated.
-     * @return the value of the metric.
-     */
-    public double apply(ColouredGraph graph);
-}
+package org.aksw.simba.lemming.metrics.single;
+
+import org.aksw.simba.lemming.ColouredGraph;
+import org.aksw.simba.lemming.metrics.Metric;
+import org.aksw.simba.lemming.metrics.single.edgemanipulation.VertexDegrees;
+import org.aksw.simba.lemming.mimicgraph.constraints.TripleBaseSingleID;
+
+/**
+ * A metric that generates a single double value.
+ * 
+ * @author Michael R&ouml;der (roeder@informatik.uni-leipzig.de)
+ *
+ */
+public interface SingleValueMetric extends Metric {
+
+    /**
+     * Applies the metric to the given graph.
+     * 
+     * @param graph
+     *            the graph for which the metric should be calculated.
+     * @return the value of the metric.
+     */
+    public double apply(ColouredGraph graph);
+    
+    
+	/**
+	 * Returns metric results that for further computations could be reused.
```

Comment:
The comment is grammatically wrong. It should be `Returns metric results that can be reused for further computations.`

Reply:
Done. I have corrected the comment.